src/components/views/groups/GroupRoomTile.js

```diff
@@ -49,6 +49,13 @@ const GroupRoomTile = React.createClass({
         });
     },
 
+    onDeleteClick: function(e) {
+        e.preventDefault();
+        e.stopPropagation();
+        this.context.matrixClient
+            .removeRoomFromGroup(this.props.groupId, this.props.groupRoom.roomId);
```

Comment:
We should probably get into the habit generally of catching errors & popping up dialogs for them, unless the plan was to get the store to do this once it was written?

Reply:
I did forget to catch and put up a dialog. I'm not sure if we can get away with the store doing it, because if we follow the same pattern as GroupSummaryStore, the store would emit an `error` event. I can see this being annoying if we want to do something following an error in the promise chain.